aws-opentelemetry-distro/src/amazon/opentelemetry/distro/_aws_metric_attribute_generator.py

```diff
@@ -423,6 +424,7 @@ def _set_remote_type_and_identifier(span: ReadableSpan, attributes: BoundedAttri
         elif is_key_present(span, AWS_BEDROCK_GUARDRAIL_ID):
             remote_resource_type = _NORMALIZED_BEDROCK_SERVICE_NAME + "::Guardrail"
             remote_resource_identifier = _escape_delimiters(span.attributes.get(AWS_BEDROCK_GUARDRAIL_ID))
```

Comment:
Same as java, we want to "extract remote_resource_identifier from cloudformation_primary_identifier to make sure the consistency of two values."? So the `AWS_BEDROCK_GUARDRAIL_ID` won't be directly used here?

Reply:
Left comment in that [PR's thread](https://github.com/aws-observability/aws-otel-java-instrumentation/pull/905/files#r1796076219).